cpp/libs/gst_nmos_plugins/README.md

```diff
@@ -0,0 +1,184 @@
+# NMOS GStreamer Plugins
+
+This repository contains GStreamer plugins that integrate AMWA NMOS functionality into your pipelines.  
+Specifically, it provides an **NMOS Audio Receiver** (`nmosaudioreceiver`), **NMOS Video Receiver** (`nmosvideoreceiver`), and **NMOS Sender** (`nmossender`) that can dynamically receive and send audio/video streams configured by the NMOS control panel.
```

Comment:
....that can receive and send **ST2110** audio/video streams....
So that we are clear that it only work for ST2110 raw audio video streams